licenseheader.txt

```diff
@@ -1,17 +1,16 @@
-  This file is part of Skript.
+This file is part of Skript.

```

Comment:
While having spaces before license is not stricly necessary, there is also no reason to change it. Or is there?

Reply:
Well, it causes a mess while looking at diffs. More lines changed than necessary. I'd almost think the license header contents were changed, not just first line (no Javadoc) and year of copyright.

But no, it does not really matter that much. People don't read diffs of license headers.